streaming/src/test/scala/org/apache/spark/streaming/UISeleniumSuite.scala

```diff
@@ -151,8 +151,9 @@ class UISeleniumSuite
         summaryText should contain ("Total delay:")
 
         findAll(cssSelector("""#batch-job-table th""")).map(_.text).toSeq should be {
-          List("Output Op Id", "Description", "Output Op Duration", "Status", "Job Id",
-            "Job Duration", "Stages: Succeeded/Total", "Tasks (for all stages): Succeeded/Total",
+          List("Output Op Id", "Description", "Output Op Duration (?)", "Status", "Job Id",
```

Comment:
Where does the "?" come from?

Reply:
"(?)" is superscript for Output Op Duration and Job Duration. This superscript got added when adding the tooltip. The same convention is followed for each page of streaming tab.